docs/hub/enterprise-hub-gating-group-collections.md

```diff
@@ -0,0 +1,166 @@
+# Gating Group Collections
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise">Enterprise Hub</a>.
+</Tip>
+
+Gating Group Collections allow organizations to grant (or reject) access to all the models and datasets in a collection at once rather than repo per repo. Users will only have to go through **a single access request**.
+
+To enable gating group in a collection:
+
+- collection owner must be an organization
+- organization must be subscribed to Enterprise Hub
+- all models and datasets in the collection must be owned by the same organization as the collection
+- all models and datasets in the collection can only be in one Gating Group Collection at once
```

Comment:
```suggestion
- each model or dataset in the collection may only belong to one Gating Group Collection, but they may be present on non-gating collections.
```

(wanted to clarify with the last clause I added, but not sure it works)

Reply:
Yes you're right, it works 👍